test/test_queries.py

```
@@ -0,0 +1,29 @@
+from pathlib import Path
+
+import pandas as pd
+
+from camvidlog2.data import EmbeddingGroup, FrameEmbedding, StringEmbedding
+from camvidlog2.queries import load_embedding_group_dataframe
+
+
+def test_strings_load_embedding_via_json_filename(tmp_path: Path) -> None:
```

Ruff (0.11.9)

Unused function argument: `tmp_path`

(ARG001)